openshift_cli_installer/utils/cluster_versions.py

```diff
@@ -86,7 +90,10 @@ def filter_versions(version, base_versions_dict, platform, stream):
     versions_dict[stream] = {version_key: {"versions": set(), "latest": ""}}
 
     for _source, versions in base_versions_dict.items():
-        if platform in (HYPERSHIFT_STR, ROSA_STR, AWS_OSD_STR) and stream != _source:
+        if (
+            platform in (HYPERSHIFT_STR, ROSA_STR, AWS_OSD_STR, GCP_OSD_STR)
```

Comment:
`(HYPERSHIFT_STR, ROSA_STR, AWS_OSD_STR, GCP_OSD_STR)` use supported_platform global variable (create it)